website/views/user.py

```
@@ -627,6 +630,48 @@ def get_context_data(self, *args, **kwargs):
 
+        # GitHub Comment Leaderboard - aggregated by commenter
+        github_comments_data = []
+        try:
+            github_comments_leaderboard = (
+                GitHubComment.objects.values("commenter_login")
+                .annotate(total_comments=Count("id"))
+                .order_by("-total_comments")[:10]
+            )
+
+            # Build leaderboard data with rank and formatted info
+            for rank, item in enumerate(github_comments_leaderboard, start=1):
```

Ruff (0.14.5)

Loop control variable `rank` not used within loop body

Rename unused `rank` to `_rank`

(B007)